tests/languages/rescript/operator_feature.test

```diff
@@ -0,0 +1,31 @@
+...
+:: :=
+= == ===
+< <= > >=
+| ^ ? '
```

Comment:
Isn't `:` also an operator because of `cond ? a : b`?

Reply:
You're right, I'll add